test/e2e/e2e_test.go

```diff
@@ -744,7 +815,8 @@ func checkClientPodRcvdUpdatedServerCert(t *testing.T, client *kubernetes.Client
 						Name:    "cert-checker",
 						Image:   "nicolaka/netshoot:latest",
 						Command: []string{"/bin/bash"},
-						Args:    []string{"-c", fmt.Sprintf("openssl s_client -showcerts -connect %s < /dev/null 2>/dev/null | openssl x509", metricsURL)},
+						Args: []string{"-c", fmt.Sprintf("openssl s_client -no-CApath -no-CAfile -CAfile /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt "+
```

Comment:
BTW note that `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt` is available; https://github.com/openshift/origin/pull/24393 suggests that it should have been removed. (Maybe https://github.com/openshift/kubernetes/pull/471 ?)

I’ve used this approach primarily to demonstrate this; I can of course change this to use ConfigMap injection instead.